mtags/src/main/scala-3/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -28,6 +29,23 @@ object MtagsEnrichments
       val p = Spans.Span(offset)
       new SourcePosition(source, p)
 
+    def localContext(params: OffsetParams): Context = {
+      val unit = driver.currentCtx.run.units.head
```

Comment:
Do we need to ever worry about the compilation units being empty here and throwing with `head`?

Reply:
In this case, we should probably throw with a more specific message. I will add it.